terraform/stack/main.tf

```diff
@@ -305,10 +305,10 @@ module "bagger" {
   cpu    = "1900"
   memory = "14000"
 
-  min_capacity = "${var.desired_bagger_count}"
+  min_capacity = "${var.desired_bagger_count - 1}"
   max_capacity = "${var.desired_bagger_count}"
 
-  desired_task_count = "${var.desired_bagger_count}"
+  desired_task_count = "${var.desired_bagger_count - 1}"
 
```

Comment:
`max_capacity = "${var.desired_bagger_count + 1}"`
looks more like the intention to me :)